custom_components/mail_and_packages/helpers.py

```diff
@@ -520,7 +522,7 @@ def fetch(
     amazon_fwds = cv.ensure_list_csv(config.get(CONF_AMAZON_FWDS))
     image_name = data[ATTR_IMAGE_NAME]
     amazon_image_name = data[ATTR_AMAZON_IMAGE]
-    amazon_days = config.get(CONF_AMAZON_DAYS)
+    amazon_days = int(config.get(CONF_AMAZON_DAYS, DEFAULT_AMAZON_DAYS))
```

Comment:
```suggestion
    amazon_days = config.get(CONF_AMAZON_DAYS, DEFAULT_AMAZON_DAYS)
```

We shouldn't need to type this to `int` it should already be an `int` from the config.
If I am wrong, please correct me. :slightly_smiling_face:

Reply:
I agree, it isn't necessarily necessary.  I can remove it. However, it would catch a case where someone may manually add : "3" in their config and this would allow it to parse correctly. 
Do you have a preference?